app/src/Routes.tsx

```diff
@@ -50,6 +51,10 @@ const router = createBrowserRouter(
         element={<ResetPasswordPage />}
         loader={resetPasswordLoader}
       />
+      <Route
+        path="/initiate-password-reset"
+        element={<InitiatePasswordResetPage />}
+      />
```

Comment:
```suggestion
      <Route
        path="/password-callback"
        element={<InitiatePasswordResetPage />}
      />
```

The word initiate seems confusing here